android/src/main/java/com/agastya/androidinappupdates/AndroidInappUpdatesModule.java

```diff
@@ -10,6 +10,7 @@
 import com.facebook.react.modules.core.DeviceEventManagerModule;
 import com.facebook.react.bridge.WritableNativeMap;
 
+import com.facebook.stetho.inspector.elements.ShadowDocument;
```

Comment:
This should not be imported.

Reply:
fixed on 73eff8d0916e79c0c3a032d223330f235b104a89